### PR TITLE
Make master releasable

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

Fix Proxy, remove user confirmation

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Locally
Storybook
Jest

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 1]
```diff
@@ -1,134 +1,131 @@
...
+var using = require("jasmine-data-provider");
 
-describe('landing page', function () {
+describe("landing page", function() {
+  beforeAll(function() {
+    browser
+      .manage()
+      .window()
+      .maximize();
+  });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]: use arrow functions
```suggestion
describe("landing page", () => {
  beforeAll(() => {
    browser
      .manage()
      .window()
      .maximize();
  });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 2]
```diff
@@ -1,134 +1,131 @@
...
 
-    beforeAll(function () {
-        browser.manage().window().maximize();
-    });
+  beforeEach(function() {
+    browser.get(process.env.URL);
+  });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
  beforeEach(() => {
    browser.get(process.env.URL);
  });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 3]
```diff
@@ -1,134 +1,131 @@
...
 
-    beforeEach(function () {
-        browser.get(process.env.URL);
+  describe("I need more Information", function() {
+    beforeEach(function() {
+      browser.get(process.env.URL);
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
  describe("I need more Information", () => {
    beforeEach(() => {
      browser.get(process.env.URL);
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 4]
```diff
@@ -1,134 +1,131 @@
...
-                expect(process.env.LP_IAMAVOLUNTEERORGANIZATION_NAVTITLE).toBe(browser.getTitle());
-            });
-
+    it("Verify if I am an employee or organization redirects to the right page", function() {
+      landingPage.needMoreInformationEmployeeOrVolunteer
+        .click()
+        .then(function() {
+          expect(process.env.LP_NEEDMOREINFORMATION_EMPORGNAVTITLE).toBe(
+            browser.getTitle()
+          );
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    it("Verify if I am an employee or organization redirects to the right page", () => {
      landingPage.needMoreInformationEmployeeOrVolunteer
        .click()
        .then(() => {
          expect(process.env.LP_NEEDMOREINFORMATION_EMPORGNAVTITLE).toBe(
            browser.getTitle()
          );
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 5]
```diff
@@ -1,134 +1,131 @@
...
-
-        landingPage.criminalRecordsReviewAct.click().then(function () {
-            expect(process.env.LP_CRIMINALRECORDSREVIEWACT_NAVTITLE).toBe(browser.getTitle());
+    it("Verify if why I need to apply for a criminal record check redirects to the right page", function() {
+      landingPage.whyINeedToApplyForACriminalRecordCheck
+        .click()
+        .then(function() {
+          expect(
+            process.env.LP_WHYINEEDTOAPPLYFORCRIMINALRECORDCHECK_NAVTITLE
+          ).toBe(browser.getTitle());
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    it("Verify if why I need to apply for a criminal record check redirects to the right page", () => {
      landingPage.whyINeedToApplyForACriminalRecordCheck
        .click()
        .then(() => {
          expect(
            process.env.LP_WHYINEEDTOAPPLYFORCRIMINALRECORDCHECK_NAVTITLE
          ).toBe(browser.getTitle());
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 6]
```diff
@@ -1,134 +1,131 @@
...
-                expect(properties.properties.lp_criminalrecordsreviewact_navtitle).toBe(browser.getTitle());
-            });
-
-        });
+    it("Verify if I'm an authorized contact redirects to the right page", function() {
+      landingPage.iAmAnAuthorizedContact.click().then(function() {
+        expect(process.env.LP_IAMANAUTHORIZEDCONTACT_NAVTITLE).toBe(
+          browser.getTitle()
+        );
+      });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    it("Verify if I'm an authorized contact redirects to the right page", () => {
      landingPage.iAmAnAuthorizedContact.click().then(() => {
        expect(process.env.LP_IAMANAUTHORIZEDCONTACT_NAVTITLE).toBe(
          browser.getTitle()
        );
      });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 7]
```diff
@@ -1,134 +1,131 @@
...
-                    browser.sleep(4000);
-                    expect(process.env.ORGVERIFICATION_URL).toBe(browser.getCurrentUrl());
-                });
+    it("Verify if I'm an employer organization redirects to the right page", function() {
+      landingPage.iAmAnEmployerOrganization.click().then(function() {
+        expect(process.env.LP_IAMANEMPLOYERORGANIZATION_NAVTITLE).toBe(
+          browser.getTitle()
+        );
+      });
+    });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    it("Verify if I'm an employer organization redirects to the right page", () => {
      landingPage.iAmAnEmployerOrganization.click().then(() => {
        expect(process.env.LP_IAMANEMPLOYERORGANIZATION_NAVTITLE).toBe(
          browser.getTitle()
        );
      });
    });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 8]
```diff
@@ -1,134 +1,131 @@
...
-            });
-        });
+    it("Verify if I'm a volunteer organization redirects to the right page", function() {
+      landingPage.iAmAnVolunteerOrganization.click().then(function() {
+        expect(process.env.LP_IAMAVOLUNTEERORGANIZATION_NAVTITLE).toBe(
+          browser.getTitle()
+        );
+      });
+    });
+  });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    it("Verify if I'm a volunteer organization redirects to the right page", () => {
      landingPage.iAmAnVolunteerOrganization.click().then(() => {
        expect(process.env.LP_IAMAVOLUNTEERORGANIZATION_NAVTITLE).toBe(
          browser.getTitle()
        );
      });
    });
  });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 9]
```diff
@@ -1,134 +1,131 @@
...
+
+  xdescribe("I need help", function() {
+    xit("Verify if criminalrecords@gov.bc.ca redirects to the right page", function() {
+      landingPage.criminalRecordsReviewAct.click().then(function() {
+        expect(properties.properties.lp_criminalrecordsreviewact_navtitle).toBe(
+          browser.getTitle()
+        );
+      });
+    });
+  });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
  it("Verify if Criminal Records Review Act (CRRA) redirects to the right page", () => {
    landingPage.criminalRecordsReviewAct.click().then(() => {
      expect(process.env.LP_CRIMINALRECORDSREVIEWACT_NAVTITLE).toBe(
        browser.getTitle()
      );
    });
  });

  xdescribe("I need help", () => {
    xit("Verify if criminalrecords@gov.bc.ca redirects to the right page", () => {
      landingPage.criminalRecordsReviewAct.click().then(() => {
        expect(properties.properties.lp_criminalrecordsreviewact_navtitle).toBe(
          browser.getTitle()
        );
      });
    });
  });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 10]
```diff
@@ -1,134 +1,131 @@
...
 
-                landingPage.accessCode.sendKeys(invalidCode);
+  describe("I am ready", function() {
+    var accessCode = require("../../input/accesscode");
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
  describe("I am ready", () => {
    let accessCode = require("../../input/accesscode");
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 11]
```diff
@@ -1,134 +1,131 @@
...
 
-                landingPage.validate.click().then(function () {
-                    browser.sleep(4000);
-                    expect(process.env.TRANSITION_URL).toBe(browser.getCurrentUrl());
-                });
+    using(accessCode.accessCode.validCode, function(validCode) {
+      it("Verify if the user is directed to the right page on validating a valid access code", function() {
+        landingPage.accessCode.sendKeys(validCode);
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    using(accessCode.accessCode.validCode, (validCode) => {
      it("Verify if the user is directed to the right page on validating a valid access code", () => {
        landingPage.accessCode.sendKeys(validCode);
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 12]
```diff
@@ -1,134 +1,131 @@
...
 
-            });
+        landingPage.validate.click().then(function() {
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
        landingPage.validate.click().then(() => {
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 13]
```diff
@@ -1,134 +1,131 @@
...
-                browser.getAllWindowHandles().then(function (windowHandle) {
-                    browser.switchTo().window(windowHandle[1]);
-                    expect(process.env.LP_VISITTHECRIMINALRECORDREVIEWWEBSITE_NAVTITLE).toBe(browser.getTitle());
-                    browser.close();
-                    browser.switchTo().window(windowHandle[0]);
-                });
-            });
+    using(accessCode.accessCode.invalidCode, function(invalidCode) {
+      it("Verify if the user is directed to the right page on validating a valid access code", function() {
+        landingPage.accessCode.sendKeys(invalidCode);
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    using(accessCode.accessCode.invalidCode, (invalidCode) => {
      it("Verify if the user is directed to the right page on validating a valid access code", () => {
        landingPage.accessCode.sendKeys(invalidCode);
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 14]
```diff
@@ -1,134 +1,131 @@
...
 
+        landingPage.validate.click().then(function() {
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
        landingPage.validate.click().then(() => {
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 15]
```diff
@@ -1,134 +1,131 @@
...
+      landingPage.visitTheCriminalRecordReviewWebsite.click().then(function() {
+        browser.getAllWindowHandles().then(function(windowHandle) {
+          browser.switchTo().window(windowHandle[1]);
+          expect(
+            process.env.LP_VISITTHECRIMINALRECORDREVIEWWEBSITE_NAVTITLE
+          ).toBe(browser.getTitle());
+          browser.close();
+          browser.switchTo().window(windowHandle[0]);
         });
+      });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    xit("Verify if visit the criminal record review website redirects to the right page", () => {
      landingPage.visitTheCriminalRecordReviewWebsite.click().then(() => {
        browser.getAllWindowHandles().then((windowHandle) => {
          browser.switchTo().window(windowHandle[1]);
          expect(
            process.env.LP_VISITTHECRIMINALRECORDREVIEWWEBSITE_NAVTITLE
          ).toBe(browser.getTitle());
          browser.close();
          browser.switchTo().window(windowHandle[0]);
        });
      });
```

File: src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js [hunk 16]
```diff
@@ -1,134 +1,131 @@
...
     });
 
-});
\ No newline at end of file
+    afterEach(function() {
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/landingpage_spec.js]:
```suggestion
    afterEach(() => {
```

File: src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js [hunk 17]
```diff
@@ -1,97 +1,105 @@
...
-var testInput = require('../../input/success');
+var testInput = require("../../input/success");
 
-var using = require('jasmine-data-provider');
+var using = require("jasmine-data-provider");
 
-describe('terms of use page', function () {
+describe("terms of use page", function() {
+  beforeEach(function() {
+    browser.get(process.env.URL);
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js]:
```suggestion
describe("terms of use page", () => {
  beforeEach(() => {
    browser.get(process.env.URL);
```

File: src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js [hunk 18]
```diff
@@ -1,97 +1,105 @@
...
 
-        browser.wait(browserWait.elementToBeClickable(orgVerificationPage.continue), 10000);
+    browser.sleep(4000);
 
-        browser.sleep(4000);
+    orgVerificationPage.continue.click();
+  });
 
-        orgVerificationPage.continue.click();
+  it("verify that the continue button is enabled when all checkboxes are checked and the terms of use is scrolled to end of section", function() {
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js]:
```suggestion
  it("verify that the continue button is enabled when all checkboxes are checked and the terms of use is scrolled to end of section", () => {
```

File: src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js [hunk 19]
```diff
@@ -1,97 +1,105 @@
...
+    browser
+      .actions()
+      .mouseMove(termsOfUsePage.termsOfUseFinalParagraph)
+      .perform();
 
-        termsOfUsePage.readAndAcceptCheckBox.click();
+    termsOfUsePage.continueButton.click();
 
-        termsOfUsePage.authorizeEmailIdCheckBox.click();
+    // bcscRedirectPage.login.click();
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js]: remove unused comment

File: src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js [hunk 20]
```diff
@@ -1,97 +1,105 @@
...
+  });
 
-        browser.actions().mouseMove(termsOfUsePage.termsOfUseFinalParagraph).perform();
+  using(
+    [
+      { unchecked: "readAndAcceptCheckBox" },
+      { unchecked: "authorizeEmailIdCheckBox" },
+      { unchecked: "termsOfUse" }
+    ],
+    function(unchecked) {
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/moduletests/termsofusepage_spec.js]:
```suggestion
    (unchecked) => {
```

File: src/ecrc-frontend/src/components/base/header/Header.js [hunk 21]
```diff
@@ -18,14 +18,14 @@ export default function Header({ header: { name } }) {
...
-            src="/images/bcid-logo-rev-en.svg"
+            src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
             width="181"
             height="44"
             alt="B.C. Government Logo"
           />
           <img
             className="img-fluid d-md-none"
-            src="/images/bcid-symbol-rev.svg"
+            src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/header/Header.js]: should this be:
```suggestion
            src="/criminalrecordcheck/images/bcid-symbol-rev.svg"
```

File: src/ecrc-frontend/src/components/base/header/__snapshots__/Header.test.js.snap [hunk 22]
```diff
@@ -16,14 +16,14 @@ exports[`Header Component Matches the snapshot 1`] = `
...
-        src="/images/bcid-logo-rev-en.svg"
+        src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
         width="181"
       />
       <img
         alt="B.C. Government Logo"
         className="img-fluid d-md-none"
         height="44"
-        src="/images/bcid-symbol-rev.svg"
+        src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/header/__snapshots__/Header.test.js.snap]: same q here

File: src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js [hunk 23]
```diff
@@ -97,52 +100,123 @@ export default function ApplicationForm({
...
+              address: { street_address, locality, region, postal_code },
+              gender,
+              given_name,
+              given_names,
+              family_name,
+              identity_assurance_level
+            }
+          } = accessJWTToken(res[0].data);
+
+          // TODO Check identity assurance level
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js]: why is this still a TODO, if we are doing it

File: src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js [hunk 24]
```diff
@@ -97,52 +100,123 @@ export default function ApplicationForm({
...
+            ["ON", "ONTARIO"],
+            ["MB", "MANITOBA"],
+            ["SK", "SASKATCHEWAN"],
+            ["YT", "YUKON"],
+            ["NT", "NORTH WEST TERRITORIES"],
+            ["NU", "NUNAVUT"]
+          ]);
+
+          let formatProvinceNm = regionMap.get(region);
+          if (formatProvinceNm === undefined) {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js]: do we also need to check `typeof undefined`?

File: src/ecrc-frontend/src/components/page/applicationForm/__snapshots__/ApplicationForm.test.js.snap [hunk 25]
```diff
@@ -16,14 +16,14 @@ exports[`ApplicationForm Component Matches the snapshot 1`] = `
...
-            src="/images/bcid-logo-rev-en.svg"
+            src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
             width="181"
           />
           <img
             alt="B.C. Government Logo"
             class="img-fluid d-md-none"
             height="44"
-            src="/images/bcid-symbol-rev.svg"
+            src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/applicationForm/__snapshots__/ApplicationForm.test.js.snap]: same q here

File: src/ecrc-frontend/src/components/page/bcscRedirect/__snapshots__/BcscRedirect.test.js.snap [hunk 26]
```diff
@@ -16,14 +16,14 @@ exports[`BcscRedirect Page Component Matches the snapshot 1`] = `
...
-            src="/images/bcid-logo-rev-en.svg"
+            src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
             width="181"
           />
           <img
             alt="B.C. Government Logo"
             class="img-fluid d-md-none"
             height="44"
-            src="/images/bcid-symbol-rev.svg"
+            src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/bcscRedirect/__snapshots__/BcscRedirect.test.js.snap]: and here

File: src/ecrc-frontend/src/components/page/consent/__snapshots__/Consent.test.js.snap [hunk 27]
```diff
@@ -17,14 +17,14 @@ exports[`Consent Page Component Matches the snapshot 1`] = `
...
-          src="/images/bcid-logo-rev-en.svg"
+          src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
           width="181"
         />
         <img
           alt="B.C. Government Logo"
           className="img-fluid d-md-none"
           height="44"
-          src="/images/bcid-symbol-rev.svg"
+          src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/consent/__snapshots__/Consent.test.js.snap]: and here

File: src/ecrc-frontend/src/components/page/error/__snapshots__/Error.test.js.snap [hunk 28]
```diff
@@ -17,14 +17,14 @@ exports[`Error Component Matches the snapshot 1`] = `
...
-          src="/images/bcid-logo-rev-en.svg"
+          src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
           width="181"
         />
         <img
           alt="B.C. Government Logo"
           className="img-fluid d-md-none"
           height="44"
-          src="/images/bcid-symbol-rev.svg"
+          src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/error/__snapshots__/Error.test.js.snap]: and here

File: src/ecrc-frontend/src/components/page/informationreview/__snapshots__/InformationReview.test.js.snap [hunk 29]
```diff
@@ -17,14 +17,14 @@ exports[`InformationReview Component Matches the snapshot 1`] = `
...
-          src="/images/bcid-logo-rev-en.svg"
+          src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
           width="181"
         />
         <img
           alt="B.C. Government Logo"
           className="img-fluid d-md-none"
           height="44"
-          src="/images/bcid-symbol-rev.svg"
+          src="/criminalrecord/check/images/bcid-symbol-rev.svg"
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/informationreview/__snapshots__/InformationReview.test.js.snap]: again

File: src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js (hunk 23 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js]: Should not be!

**Pull request merged**